website/moneybirdsynchronization/services.py

```diff
@@ -322,6 +418,36 @@ def _sync_sales_orders():
     _try_create_or_update_external_invoices(sales_orders)
 
 
+def _sync_merchandise_sales():
+    """Create journals for sales orders that are merchandise sales."""
+    merchandise_sales = Order.objects.filter(
```

Comment:
I would say don't filter on the shift title, but on the product being a merchandise product

Reply:
Doest filtering on 

```
        items__in=MerchandiseItem.objects.all(),

```

work

Reply:
Since `items